### ISSUE TITLE
nbextensions not appearing in notebook 

### ISSUE THREAD

**Issue by zozoheir** [Jupyter-contrib/jupyter_nbextensions_configurator#89]
Hi everyone,

As the title says, the extensions do not appear in the notebook. I am using Conda 4.5.11 and Python 2.7.13. I simply ran this:
conda install -c conda-forge jupyter_nbextensions_configurator  as mentionned on the main page.

After that, I am able to see the NBextensions tab in my "tree" view. I am also able to see all the extensions, Enable/Disable them as I want, **but not chance will appear in any of my notebooks, even after restarting jupyter**
I tried to reinstall NBextensions, reclose and reopent jupyter notebook, but same issue...

I couldn't find anything on Google or github. Hope someone can help here...
Thanks in advance,

Othmane

**Comment by juhasch** [comment]
Yes, I need to update conda-forge to include a required fix. I will do so in the next days.

**Comment by ghuname** [comment]
I have the same problem with jupyter lab 0.35.4 and python 3.7.
On link http://localhost:8888/nbextensions I can see all extensions, turn them on/off, but they don't appear in the notebook.

**Comment by pramitchoudhary** [comment]
@ghuname Jupyter Notebook is a web interface to a computational environment. However, JupyterLab could be thought of like an IDE(Integrated Development Environment). 
For a quick reference check out: https://stackoverflow.com/questions/50982686/what-is-the-difference-between-jupyter-nootbook-and-jupyter-lab

**Comment by ghuname** [comment]
@pramitchoudhary  Can I use https://jupyter-contrib-nbextensions.readthedocs.io/en/latest/nbextensions/execute_time/readme.html with jupyterlab?
